src/main/kotlin/ratpack/example/kotlin/Helpers.kt

```diff
@@ -0,0 +1,41 @@
+@file:Suppress("Unused")
+
+/*
+ * Helper functions to make the example more Kotlin like. All of them are extension functions that add
+ * syntactic sugar to Ratpack's methods.
+ *
+ * The names are prefixed by 'k' because the signature of some methods is the same that Ratpack's ones and
```

Comment:
I'm not a Kotlin expert but I'm not a huge fan of 'k' prefix.  What you're demonstrating is great, effectively a Kotlin DSL for Ratpack, but I wonder whether it belongs in this example.  If you were going to use Kotlin with Ratpack today, is this what you would do idiomatically?

@robfletcher do you have an opinion here?